### DISCUSSION TITLE
How to change spell's duration, stats etc?

### DISCUSSION THREAD

**Discussion by IchorZ** [TrinityCore/TrinityCore#29312] [Q&A] [answered]
I was able to change the cooldown time of some abilities (racial), but there was a problem with the troll ability (26297). I want to reduce its cd and duration. I was able to change the CD (I changed the data both on the server and on the server), but I can't figure out how to change the duration of its action in any way. I didn't find any information anywhere. They wrote that in the Spell table.dbc has a DurationIndex column, which in theory should have the duration of the spell effect. But it costs 1 (both in the client and on the server), although in the game it lasts 10 seconds. Please help me)
![image](https://github.com/TrinityCore/TrinityCore/assets/144030004/00dba1b9-79b5-484a-825b-5993662b01d7)
![image](https://github.com/TrinityCore/TrinityCore/assets/144030004/355fd2d0-1c80-485e-a6a2-f7bb47c0a535)
![image](https://github.com/TrinityCore/TrinityCore/assets/144030004/3ecc4403-9f82-4373-948a-267efca2b4d5)

**Comment by IchorZ** [comment]
well i found that "1" here is a INDEX(yes) of duration. so index 1 means 10 sec, index 32 means 6 sec etc. But i didn't found how to modify this percentage of "20%". helpppp

**Comment by IchorZ** [comment]
solved by the "method of scientific poke".

**Reply by Faq** [reply]
You could answer your own question, so others who need this can find this.

**Reply by shamage** [reply]
Dear author, please provide an answer as to how you solved it - it would be instructive for me and others. Well thank you .

**Reply by IchorZ** [reply, marked as answer]
To modify percentage ore power of any skill you should modify column EffectBasePoints_1(in case of spell that i modified in this post). This column stores int values. Specifically, this column stores information about the strength of the FIRST spell effect. There may be three of them in total. In addition, the value in this column should be written 1 lower than you need. For example, I changed the percentage of speed change from 20 to 15. So, there was a value of 19 in this column, which I changed to 14.

UPD: if you need, then all the information about this dbc as well as any others is on the wowdev website. Here is a link specifically for this table: https://wowdev.wiki/DB/Spell
